AppStore/German/Description.iOS.txt

```diff
@@ -0,0 +1,29 @@
+Hast du dich jemals gefragt, "Wer ist dieser Schauspieler?", oder vielleicht "Wann wurde dieser Film veröffentlicht?", oder "Wie war der Titel dieser einen Episode meiner Lieblingsserie?" Vielleicht bist du ein Filmkenner und fragst dich, "Wie alt war diese Person, als dieser Film veröffentlicht wurde?"
+
+Wolltest du schon immer solche Fragen beantworten können, *ohne* alle fünf Sekunden irgendwo aufgefordert zu werden dich anzumelden? Ohne Werbung und ohne automatisch abgespielte Videos? Ohne auf ein multinationales Konglomerat angewiesen zu sein, das dir Dinge verkaufen möchte?
+
+Dann wirst du Callsheet lieben.
+
+Callsheet ist der beste Weg, um schnell Informationen über Besetzung und Crew in Fernsehserien und Filmen zu finden. Es geht mit deiner Zeit respektvoll um, versucht nicht, dir Dinge zu verkaufen, die du nicht brauchst, und ist darauf ausgelegt, deine Fragen schnell und unkompliziert zu beantworten.
+
+Callsheet bietet auch einige neuartige Funktionen, die deine bisherige Film-/Seriendatenbank-App nicht hat:
+
+• Spoiler vermeiden beim Anschauen von TV-Serien, indem optional Charakternamen, Episodenzahlen, Episodentitel oder Episodenvorschaubilder ausgeblendet werden.
```

Comment:
Looking at this long-form description I'm starting to feel like "TV-Serien" might be better than "TV-Sendungen" in general.

Reply:
It’s a bit murky: the main use case is indeed TV series, but there are also other shows in there, like "unscripted" "reality" TV ;)

Reply:
Hence, for the I’m a Celebrity Get Me out of Here crowd, I left it TV-Sendungen in the UI. The quoted passage from the store description is more about TV series, so I translated it TV-Serien there

Reply:
That makes sense.